stdlib/2/unittest.pyi

```diff
@@ -15,6 +15,10 @@ _FT = TypeVar('_FT')
 _ExceptionType = Union[Type[BaseException], Tuple[Type[BaseException], ...]]
 _Regexp = Union[Text, Pattern[Text]]
 
+_SysExcInfoType = Tuple[Optional[Type[BaseException]],
+                        Optional[BaseException],
+                        Optional[types.TracebackType]]
+
 class Testable(metaclass=ABCMeta):
```

Comment:
Do we even need this class? It does not exist in the implementation and I don't really know what purpose it serves in the stubs, especially now that you removed its uses.

Reply:
It effectively means `Union[TestCase, TestSuite]`. I didn't remove all uses, just the ones that I noticed should be `TestCase` instead.

It shouldn't be a class, but changing that would take a lot more work.